### ISSUE TITLE
Question - How can we use SVG ?

### ISSUE THREAD

**Issue by AntoninMartin** [jonathanpeppers/glidex#33]
Hi Jonathan,

Thanks for this package ... I've been using FFImageLoading for a long time but many bugs with recents version of Xamarin Forms discouraged me.

I would try your package in my app, but can I use svg files and how can I configure glidex to work like this ?

Thanks :)

**Comment by jonathanpeppers** [comment]
Looks like Glide has a Java sample using SVG files [here](https://github.com/bumptech/glide/tree/master/samples/svg/src/main/java/com/bumptech/glide/samples/svg).

But it looks like you have to write a bunch of code to make it work...

Parts of Glide on the Java side use [compile-time annotations](http://hannesdorfmann.com/annotation-processing/annotationprocessing101), so I'm not sure there is a simple way to set this up with Xamarin.Android. We would have to add a feature to enable `javac` calls to process custom annotations.

If I have time, I will see if there is a way to make this work without using annotations.

**Comment by themronion** [comment]
@jonathanpeppers now that Xamarin introduced paths maybe it can somehow be achieved?

**Comment by jonathanpeppers** [comment]
@themronion can you share a link of what you are referring to?

Xamarin.Android does not have Java annotation support.

**Comment by themronion** [comment]
@jonathanpeppers yeah, sorry, i think it is possible somehow only for xamarin.forms. Here is what i was referring too: [click](https://javiersuarezruiz.wordpress.com/2020/06/30/xamarin-forms-tips-and-tricks-working-with-shapes/)

**Comment by jonathanpeppers** [comment]
Ok yeah, @themronion this looks like it is using `<Path/>` that came with the new shapes features in Xamarin.Forms 4.8.

I would use `<Path/>` directly for an `.svg` file, that seems like a reasonable choice. glidex could be useful at some point if you needed to _download_ `.svg` files from the internet, but they can also be pretty small in file size compared to other image types.
